chapters/part3/fairness.md

```diff
@@ -0,0 +1,733 @@
+
+## Biases in Models
+
+You may often hear, in the context of biases, "It's not the model, it's the data!" This is far from the truth. Models are excellent at <em>encoding</em> and <em>amplifying</em> biases. 
+
+1. **Imbalanced datasets:** Model performance inherently favors groups with rich features and a large, diverse sample, thereby marginalizing groups with limited features and a poor sample size. 
+
+2. **Spurious correlations:** Models learn complex, non-linear and spurious correlations between features and annotations that often reflect social biases and are difficult to detect and eliminate. For example, [Wang et al.](https://api.semanticscholar.org/CorpusID:195847929) discovered that even after removing subjects from images, a model could still easily predict the gender of subjects because women were photographed overwhelmingly frequently in the kitchen, that is, the model leveraged the existence of a kitchen in the image as a proxy for gender.
+
+3. **Contextualization:** LSTMs and attention-based models, through contextualization, allow for biases to be propagated across words in a sentence.
+
+Balanced datasets are not effective at taming bias amplification because of latent variables.
+
+## Biases in Interpretation
+
+Beyond biases in data and models, there also exist biases in interpretation [2]:
+
+1. **Confirmation bias:** The tendency to search for, interpret, favor, recall information in a way that confirms preexisting beliefs, e.g. researchers misconstrue the ability of a model to predict binary gender as the non-existence of non-binary genders
+
+2. **Overgeneralization:** Coming to conclusions based on information that is too general, e.g. researchers conclude that a model works well for everyone based on its overall accuracy on data containing mostly white people
+
+3. **Correlation fallacy:** Confusing correlation with causation
+
+4. **Automation bias:** Propensity for humans to favor suggestions from automated decision-making systems over contradictory information without automation
+
+Biases in interpretation truly necessitate the creation and usage of metrics that can uncover unfairness and biased associations in models. 
+
+## Call to Action
+
+Why should we care about fairness and biases? Because ensuring fairness and mitigating biases are inherently normative [1]. As language models become more prevalent in critical decision-making systems, their unfair encoding and amplification of biases can pose serious harms for already-marginalized communities. In NLP literature, harms are often categorized into the following (Barocas and Selbst, 2016):
```

Comment:
I had to look up "normative" to understand this sentence.  I'm not sure that sentence is even necessary and I might remove it, as the following sentence is clearer.  Or maybe something like "Because models are increasingly used throughout everyday life and increasingly have a real impact on all people."